alpha/declcfg/declcfg_to_model.go

```diff
@@ -165,7 +166,8 @@ func ConvertToModel(cfg DeclarativeConfig) (model.Model, error) {
 	}
 
 	if err := mpkgs.Validate(); err != nil {
-		return nil, err
+		fmt.Println("Joe - ignore errors in validation for testing")
+		//return nil, err
```

Comment:
Is this still needed?

Reply:
oops,  no, we can clean that up once we get concurrence that we want to move all this into oc-mirror directly, I will add this to the olm call tomorrow if room on the agenda